src/main/java/Task.java

```diff
@@ -0,0 +1,31 @@
+public class Task {
+
+    public String name;
+    public Boolean status;
+
+
+    public Task(String name){
+        this.name = name;
+        this.status = false;
+    }
+
+
+    public void setStatus(String action){
+        if (action.equals("mark")){
+            this.status = true;
+        } else if (action.equals("unmark")) {
+            this.status = false;
+        }
+    }
+
+
+    public String toString(){
+        String checkbox = "[ ]";
+        String typeIndicator = null;
+        if(status){
+            checkbox = "[X]";
+        }
+
+        return  checkbox + " " + name;
+    }
```

Comment:
This method may be unused. Maybe can use the 'super.toString()' method in the child class to call this method. 
For example, Deadline.java Line 15:
'return "[D]" + checkbox + " " + name + " (by:" + deadline + ")";' 
can be written as: 
'return "[D]" + super.toString() + " (by:" + deadline + ")";'